### DISCUSSION TITLE
The name of the software

### DISCUSSION THREAD

**Discussion by marrus-sh** [cat-KIBI-glitch/mastodon#11] [Request for Comments]
Right now the software is called **Kibicat Mastodon**, because it is being developed by the Kibicat Organization (here on GitHub) and it is a Mastodon‐compatible fediverse server. However, the name “Mastodon” has strong associations with Mastodon gGmbH (the legal entity which controls upstream Mastodon development), and while I don’t *anticipate* problems, giving ourselves a name which is perhaps not‐so‐blatantly‐associated might be a good thing should this fork ever acquire adoption beyond simply the `glitch.cat.family` server (aka if we ever develop useful new features which people might want).

So this thread exists to bikeshed alternative names! (Or to vouch support for the existing one.)

**Comment by aescling** [comment]
i don’t have any ideas, but i *much* support bikeshedding for a new name

**Comment by marrus-sh** [comment]
I mean the name “glitch” is problematic for “being an extremely common word that programmers love” reasons, but if we wanted to, I don’t think anybody could stop us from laying claim to GlitchSoc or a derivative.

The main reasons I don’t like GlitchSoc as a name are:—

1. There is an instance also called GlitchSoc. This gets confusing fast and is one of the reasons why I insisted on not calling the software GlitchCat in the README.

2. GlitchSoc commonly refers to Mastodon Glitch Edition, which is a separate fork.

3. If the name is too long, people will clip it, shortening “Kibicat GlitchSoc” to something like “KibiGlitch” (dropping cat). That’s not ideal, so we should have a software name which is unambiguous without the “Kibicat” designation.

On the other hand:—

1. Kibicat Mastodon is technically a fork of GlitchSoc, not Mastodon.

2. Kibicat Mastodon and GlitchSoc share a notable developer (me).

3. (minor note) I’m not sure how prevalent the spelling GlitchSoc actually *is*, compared to “Glitch” or “Glitch Social”.

4. (awkward circumstance) The name GlitchCat makes a lot less sense if we get rid of the Glitch branding entirely.

So we might want to consider a Glitch name which is not GlitchSoc (or GlitchCat). Like NewGlitch, (MewGlitch,) GlitchTwo, (GlitchU,)…

Just some ideas!

**Reply by marrus-sh** [reply]
(I actually really like MewGlitch, except for the fact that literally everyone who sees it won’t think “sound a cat makes which rhymes with new” + “glitch social”, but instead think it is a meaningless nerdy reference to <https://bulbapedia.bulbagarden.net/wiki/Mew_glitch>. We could maybe get away with a more “nya” spelling of new, like “nyu”?)

**Reply by aescling** [reply]
as an aside, one slightly unfortunate thing about having called our instance GlitchCat is there is a twitch streamer (with a patreon, no less!) going by that name already lol

i like the name too much (and find it too sensible) to really want to change it; but we might want to prepare to have to bikeshed new names for the instance itself if it comes to that

**Reply by marrus-sh** [reply]
(I'm mostly okay with instance names not being particularly unique; GlitchCat in particular is probably a very common name because like. Glitch + Cat. But it's usually obvious in context when you mean the instance.

Donphan is the name of a pokémon but there's never been any problem also calling the instance Donphan. The only reason I would think to change it would be if someone else using the name GlitchCat became famous enough that people might think we were a fan-instance. Don't think that's a concern right now.)

**Comment by marrus-sh** [comment]
Meow‐variations of NewGlitch which might be acceptable :—

+ MiuGlitch
    + Jyutping for 喵
    + Basically a spelling of “mew” which people won’t associate with the pokémon
+ NyuGlitch
    + Literally Japanese for “new”: <https://en.wiktionary.org/wiki/ニュー#Japanese>
    + Also Korean: <https://en.wiktionary.org/wiki/뉴#Korean>
    + Some languages associate it with “gnu” instead though
+ NyewGlitch
    + Basic Latin spelling of “ñew”: <https://en.wiktionary.org/wiki/ñew#Spanish>
    + But this is a more cumbersome spelling for English speakers
